inspire/utils/latex.py

```diff
@@ -0,0 +1,428 @@
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2015 CERN.
+##
+# INSPIRE is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+##
+# INSPIRE is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+##
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#
+
+import re
+from invenio.modules.knowledge.api import get_kbr_keys
+
+
+class MissingRequiredFieldError(LookupError):
+
+    """Base class for exceptions in this module.
+    The exception should be raised when the specific,
+    required field doesn't exist in the record.
+    """
+
+    def _init_(self, field):
+        self.field = field
+
+    def _str_(self):
+        return "Missing field: " + self.field
+
+
+class Latex(object):
+    """
+    TODO Fix the citation number latex
+    e.g %245 citations counted in INSPIRE as of 21 Aug 2015
+    """
+    """docstring for Latex"""
```

Comment:
This is quite generic msg. You can just have one docstring where you explain that this class is used to output Latex format and also the `TODO`